### PR TITLE
feat: tweak env switcher

### PR DESCRIPTION
**Pull request by rofe**
- [x] disable current env in picker
- [x] hide live env if prod env exists
- [x] no env switcher in admin

### CODE REVIEW

File: src/extension/app/components/action-bar/menu-item/menu-item.css.js [hunk 1]
```diff
@@ -24,6 +24,13 @@ export const style = css`
     font-weight: 700;
   }
 
+  :host(.current-env.env-edit[aria-disabled="true"]) #label,
+  :host(.current-env.env-edit[disabled]) #label,
+  :host(.current-env.env-edit[aria-disabled="true"]) [name="description"]::slotted(*),
+  :host(.current-env.env-edit[disabled]) [name="description"]::slotted(*) {
+    color: var(--mod-menu-item-label-content-color-default);
```
Comment (dylandepass) [inline src/extension/app/components/action-bar/menu-item/menu-item.css.js:31]: @rofe Curious where this variable comes from?

**Review by dylandepass — APPROVED** [review]

File: src/extension/app/components/action-bar/menu-item/menu-item.css.js (hunk 1 above)
Comment (rofe) [reply src/extension/app/components/action-bar/menu-item/menu-item.css.js:31]: @dylandepass Read them out of spectrum CSS 😬 happy to change to something more appropriate of course

Comment (rofe) [reply src/extension/app/components/action-bar/menu-item/menu-item.css.js]: removed the `mod-` prefix, looks the same without

**Pull request merged**